vocab/properties.json

```diff
@@ -147,15 +147,15 @@
         ]
     },
     "associatedFileExtension": {
-        "description": null,
+        "description": "String of characters attached as suffix to the names of files of a particular format.",
         "name": "Associated file extension",
         "sameAs": null,
         "schemas": [
             "core/schemas/v3/data/contentType.schema.tpl.json"
         ]
     },
     "author": {
-        "description": null,
+        "description": "Writer or creator of a literary or creative work.",
```

Comment:
don't we also use it for contributors to a Dataset? (not a literary work, only a creative work in a broad sense)

Reply:
How about adding "... Also includes creators of a dataset publication." or "Writer or creator of a literary or creative work, as well as a dataset publication." ?

Reply:
https://schema.org/Dataset is listed under "creativeWork". I don't think that we need to specifically mention datasets here.